README.md

```diff
@@ -1,89 +1,134 @@
 
-You can check the logs of the connector with:
+1. Create and configure a [configuration file](#configuration) ensuring you have set at least the basic properties.
 
-```
-docker-compose logs connector
-```
+2. Start the cluster using:
 
-To check the connector is working, subscribe to messages on the `kafka-connect-ably-example`
-Ably channel in a separate terminal window (this uses [SSE](https://ably.com/documentation/sse)
-for convenience, but you could also use an [Ably SDK](https://ably.com/download)):
+    `docker-compose up -d`
 
-```
-curl -s -u "<your Ably API key>" "https://realtime.ably.io/sse?channel=kafka-connect-ably-example&v=1.1"
-```
+    **Note**: You can view the logs using `docker-compose logs connector`
 
-Produce some messages to the `kafka-connect-ably-example` Kafka topic using the
-`kafka-console-producer` CLI tool in the Kafka container:
+3. Once the pods have started, you can test the connector by subscribing to your Ably channel using [SSE](https://ably.com/documentation/sse) in a new terminal window. Replace `<channel-name>` with the channel set in your configuration file and `<ably-api-key>` with an API key with the capability to subscribe to the channel.
 
-```
-docker-compose exec -T kafka kafka-console-producer --topic kafka-connect-ably-example --broker-list kafka:9092 <<EOF
-message 1
-message 2
-message 3
-EOF
-```
+    `curl -s -u "<ably-api-key>" "https://realtime.ably.io/sse?channel=<channel-name>&v=1.1"`
 
-The messages should have been published to Ably by the connector and appear on
-the SSE subscription in your other terminal window:
+    **Note**: SSE is only used as an example. An Ably SDK can also be used to subscribe to the channel.
 
-```
-id: e026fVvywAz6Il@1623496744539-0
-event: message
-data: {"id":"1543960661:0:0","clientId":"kafka-connect-ably-example","connectionId":"SuJTceISnT","timestamp":1623496744538,"encoding":"base64","channel":"kafka-connect-ably-example","data":"bWVzc2FnZSAx","name":"sink"}
+4. Produce a set of test messages in Kafka using the Kafka CLI tool. Replace `<kafka-topic-name>` with one of the topics set in your configuration file.
 
-id: e026fVvywAz6Il@1623496744539-1
-event: message
-data: {"id":"1543960661:0:1","clientId":"kafka-connect-ably-example","connectionId":"SuJTceISnT","timestamp":1623496744538,"encoding":"base64","channel":"kafka-connect-ably-example","data":"bWVzc2FnZSAy","name":"sink"}
+    ```
+    docker-compose exec -T kafka kafka-console-producer --topic <kafka-topic-name> --broker-list kafka:9092 <<EOF
+    message 1
+    message 2
+    message 3
+    EOF
+    ```
 
-id: e026fVvywAz6Il@1623496744539-2
-event: message
-data: {"id":"1543960661:0:2","clientId":"kafka-connect-ably-example","connectionId":"SuJTceISnT","timestamp":1623496744538,"encoding":"base64","channel":"kafka-connect-ably-example","data":"bWVzc2FnZSAz","name":"sink"}
-```
+5. In the terminal window where you subscribed to the Ably channel, you will receive the following messages:
```

Comment:
Probably say "you will receive messages similar to the following" (since the messages they get will be different)

Reply:
Updated this too, thanks.